packages/components-react/projects/remix/app/routes/_index.tsx

```diff
@@ -13,6 +13,11 @@ export default function _index(): JSX.Element {
         <li>
           <Link to="/accordion-layout-shift">Accordion Layout Shift</Link>
         </li>
+        <li>
+          <PLink>
+            <Link preventScrollReset={true} to="/modal-route-change">Modal Route Change</Link>
```

Comment:
Perhaps the page and url should be called something like ModalPage because it is clear that a page is routable.